src/main/java/se/michaelthelin/spotify/SpotifyApi.java

```diff
@@ -98,10 +98,6 @@ public class SpotifyApi {
   private final String scheme;
   private final String host;
   private final Integer port;
-  private final String proxyUrl;
-  private final Integer proxyPort;
-  private final Integer proxyUsername;
-  private final Integer proxyPassword;
```

Comment:
Why delete those? Isn't that a breaking change?

Reply:
Sorry, my mistake, I should have only included the first commit in the pull request.